packages/protocol/contracts/authorizers/MetaAdminnable.sol

```diff
@@ -41,7 +41,7 @@ contract MetaAdminnable is RankedAdminnable, IMetaAdminnable {
         override(RankedAdminnable, IRankedAdminnable)
         returns (uint256)
     {
-        if (msg.sender == metaAdmin) return MAX_RANK;
+        if (admin == metaAdmin) return MAX_RANK;
```

Comment:
Here's a bug

Reply:
I actually mentioned this a while back https://github.com/api3dao/airnode/pull/409#discussion_r664677178 thought it was intended.

Reply:
Right, I missed that comment